src/Game/main.cpp

```diff
@@ -136,7 +155,24 @@ int main() {
     
     glfwTerminate();
 
+    int returnValue = 0;
+
+    if ( maxRamUsed > maxRamLimit ) {
+        Log() << "Ram limit violated.";
```

Comment:
Missing newline in output message. Also applies to the upcoming two logging statements.